02_munge/src/munge_usgs.py

```diff
@@ -40,8 +40,14 @@ def process_data_to_csv(raw_datafiles, flags_to_drop):
         cols = df.columns.drop('datetime')
         df[cols] = df[cols].apply(pd.to_numeric, errors='coerce')
 
-        # (to be added in future) aggregate data to specific timestep?
-        # df = df.groupby([df['datetime'].dt.date])[cols].mean()
+        # aggregate data to daily timestep
```

Comment:
Given what Salme said today, it might make sense to make our aggregation time scale (i.e. daily, hourly etc) a variable if possible

Reply:
Good point. I've created a variable to specify this. The way I set it up, we have to add a little code for each timestep we might want to process, so right now 'daily' is the only option. I can add more options as the need arises.

There might be a more sleek way to do this, but I'll dig into that when we add more timestep processing levels. For now, at least it draws attention to the fact that this is a variable we might need to change. Let me know if you think this works for now!